sdk/resources/Microsoft.Azure.Management.Resource/tests/ScenarioTests/TemplateSpecsTests.ScenarioTests.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ResourceGroups.Tests
+
+{
+    class LiveTemplateSpecsTest : TestBase
```

Comment:
>class LiveTemplateSpecsTest : TestBase [](start = 4, length = 38)

We should probably communicate with @StuartKo and add Template Spec tests.

Reply:
I had created this just as a placeholder of sorts. I've already talked to him about tests.